docs/resources/aws_servicecatalog_launch_role_constraints.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About the aws_servicecatalog_launch_role_constraints Resource
+platform: aws
+---
+
+# aws\_servicecatalog\_launch\_role\_constraints
+
+Use the `aws_servicecatalog_launch_role_constraints` InSpec audit resource to test properties of multiple AWS Service Catalog launch constraint.
+
+## Syntax
+
+Ensure that a portfolio exists.
+
+    describe aws_servicecatalog_launch_role_constraints(portfolio_id: 'PORTFOLIO_ID') do
+      it { should exist }
+    end
+
+## Parameters
+
+`portfolio_id` _(required)_
```

Comment:
Is `porfolio_id` correct, or is it just `id`?

Reply:
id is the correct parameter.